scripts/setup_linux.sh

```diff
@@ -102,6 +102,11 @@ print_warning() {
   printf "\e[0;33m%s\e[0m" "${1}"
 }
 
+# Verify this script is running from the correct folder (root directory)
+([ -e setup.py ] && (more setup.py | grep -q "name='rlgarage'")) \
+  || _PRINT_HELP=yes die \
+  "This setup script must be run from the root garage directory." 1
```

Comment:
Can you make this message more helpful? If a user hits this error, they might be confused if they only got this message.

For instance, there's a path in the repo which is `garage/garage`--which do you mean? Tell them to go to the root directory of the git repository.

I think it's even best to provide a snippet they can run, e.g. `./scripts/setup_linux.sh`, which will only work if they're in the right directory.